default.nix

```diff
@@ -186,6 +186,8 @@ let
             text = "Combined documentation for all the public Plutus libraries.";
         };
       };
+
+      marlowe-tutorial = pkgs.callPackage ./marlowe-tutorial/doc { asciidoctor = asciidoctorWithEpub3; };
```

Comment:
I guess the marlowe tutorials aren't compiled with lhs? Shame.

Reply:
they will be coming I think

Reply:
but not in this PR...